### ISSUE TITLE
Changing TREEVIEW_WIDTH has no effect (Origin: bugzilla #681831)

### ISSUE THREAD

**Issue by doxygen** [doxygen/doxygen#4849]
status RESOLVED severity *normal* in component *doxywizard* for *---*
Reported in version *1.8.1.2* on platform *Other*
Assigned to: Dimitri van Heesch

On 2012-08-14 12:24:48 +0000, Sean wrote:

> I have enabled the option to generate a tree view. A pane is created on the left side of my documentation as expected. The pane is too wide for my taste, so I changed TREEVIEW_WIDTH from its default value (250) to a smaller one (125). I regenerated the documentation, but there was no apparent difference in the html. 
> 
> As a sanity check, I disabled the treeview and ran doxygen again. This time there was no treeview. I then enabled it again (with TREEVIEW_WIDTH still set to 125) and ran doxygen again. The treeview was back, but was still the same width it always is. 
> 
> I could find no other bugs relating to TREEVIEW_WIDTH. That might explain why this has not been spotted before.

On 2012-08-15 18:15:23 +0000, Dimitri van Heesch wrote:

> Confirmed. Should be fixed in the next subversion update.

On 2012-12-26 16:09:22 +0000, Dimitri van Heesch wrote:

> This bug was previously marked ASSIGNED, which means it should be fixed in
> doxygen version 1.8.3. Please verify if this is indeed the case. Reopen the
> bug if you think it is not fixed and please include any additional information
> that you think can be relevant.

**Comment by farr64** [comment]
Apparently, this bug still exists in 1.8.17.

**Comment by doxygen** [comment]
@farr64 Are you aware that the width is stored in a `doxygen_width` cookie in the browser? (this is done to prevent width change every time the user navigates to another doxygen generated page). So check if deleting the cookies helps. If not then please a attach a self contained example (e.g. minimal source + config file in a zip or tar).

**Comment by farr64** [comment]
Could you please share the name of the cookie? That's the first thing I tried, but I could not find it (at least not obviously).

Thanks.

**Comment by albert-github** [comment]
@farr64 The name of the cookie is as indicated `doxygen_width`.
I don't know which browser you are using but for Firefox a starting point can be https://superuser.com/questions/392176/view-cookies-in-mozilla-firefox.
(I found this by googling for: firefox list cookies).

**Comment by farr64** [comment]
Found it.
Thanks @albert-github 

After a little digging, I found this for macOS Safari, which allows you to see the contents of all local storage( including cookies) for your current web page.

On the Safari menu on top:

Develop => Show Web Inspector => Click on the "Storage" tab

If you don't see the "Develop" Safari menu item, open Safari preferences and check "Show Develop menu in menu bar" at the bottom of the "Advanced" pane.

Enjoy
